litmus-portal/frontend/src/views/CreateWorkflow/ReliabilityScore/__tests__/ReliabilityScore.test.tsx

```diff
@@ -0,0 +1,23 @@
+import '@testing-library/jest-dom';
+import { act, screen } from '@testing-library/react';
+import React from 'react';
+import { render } from '../../../../testHelpers/test-util';
+import ReliabilityScore from '../index';
+
+describe('Reliability Score', () => {
+  it('has the right slider properties', async () => {
+    const ref = React.createRef();
+    act(() => {
+      render(<ReliabilityScore ref={ref} />);
+    });
+
+    await act(async () => {
+      await expect(screen.getByRole('slider').className).toBe(
+        'MuiSlider-thumb WithStyles(ForwardRef(Slider))-thumb-47 MuiSlider-thumbColorPrimary PrivateValueLabel-thumb-54'
```

Comment:
This generated class name will always be the same?

Reply:
Yes the test specifically only creates one slider to have the generated styles the same always